cycle_en_terre_website_design/static/src/less/cycle_en_terre_website_design.less

```diff
@@ -0,0 +1,141 @@
+@import url('https://fonts.googleapis.com/css?family=Oswald');
+@import url('https://fonts.googleapis.com/css?family=Old+Standard+TT');
+
+body {
+    font-size: 16px;
+    line-height: 25px;
+    font-weight: 500;
+    letter-spacing: 0px;
+    color: #626262;
+    font-family: Old Standard TT, Arial, Tahoma, sans-serif;
+}
+
+a {
+    color: #f0b61c;
+}
+
+a:hover {
+    text-decoration:underline;
+    color: #a67c0a;
+}
+
+a, a:visited, a:focus {
+    text-decoration: none;
+    outline: 0;
+    }
+
+#wrap {
+   padding-top:60px;
+   padding-bottom:80px;
+}
+
+.navbar-default {
+    background-color:inherit;
+}
+
+.navbar-right {
+    float: none;
+    margin-right: 0;
+}
+
+@media (min-width: 768px) {
+    .navbar-right {
+        float: none !important;
+        margin-right: 0;
+    }
+}
+
+#top_menu li a b  {
+    font-weight:normal;
+}
+
+#top_menu {
+     text-transform:uppercase;
+     background:inherit;
+     font-size:18px;
+     font-family: Oswald, Arial, Tahoma, sans-serif;
+     font-weight: 400;
+     color:#444;
+	}
+
+.navbar-default .navbar-nav li > a {
+    color:#444;
+    background:inherit;
+    margin:0;
+    padding: 15px 0;
+    text-align:center;
+}
+
+.navbar-default .navbar-nav > .active > a, .navbar-default .navbar-nav > .active > a:hover, .navbar-default .navbar-nav > .active > a:focus,
+ .navbar-default .navbar-nav > .open > a, .navbar-default .navbar-nav > .open > a:hover, .navbar-default .navbar-nav > .open > a:focus {
+    color: #f0b61c;
+    background-color: #fff;
+    border:none;
+}
+
+.navbar-default .navbar-nav li > a > span  {
+    border-right: 1px solid rgba(0,0,0,.05);
+    padding: 0 20px ;
+    display: block;
+    margin:0 auto;
+}
+
+.navbar.navbar-static-top ul.nav > li.divider {
+   border-right:0px;
+   padding: 0 10px ;
+}
+
+.navbar-right .dropdown-menu  {
+    box-shadow: 0 0 0 0 transparent !important;
+    border:none;
+    background:#f0b61c;
+    text-transform: normal;
+    border-radius:0;
+    padding:0;
+    left: -10px;
+    right: auto;
+}
+
+.navbar-default .navbar-nav li > .dropdown-menu a {
+    color: #fff;
+}
+
+.navbar-default .navbar-nav li > ul > li:hover {
+    background-color: #e2ab19;
+}
+
+
+.navbar-default .navbar-nav li > .dropdown-menu a:hover {
+    color: #444;
+    background-color: #e2ab19;
+}
+
+footer {
+   background-color: #f0b61c;
+   color:#fff;
+}
+
+footer > div {
+   border-spacing:10px;
+}
+
+footer a {
+    color:#000;
+}
+
+footer a:hover, footer a:focus, footer a:visited {
+    color:#fff;
+}
+
+div.bg-img-footer {
+    background: #f0b61c url("/cycle_en_terre_website_design/static/src/img/vague-orange.jpg") no-repeat center top ;
+    max-width: 1924px;
+    height : 74px;
+    margin: -34px auto 0 auto;
+    }
+
```

Comment:
Remove empty lines. :)

Reply:
Dou you mean all the empty lines in the .less file ?

Reply:
I mean all empty lines at the end of the file. :)

Reply:
ok, done :)